add-domain

```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
+steamcomnnulty.com
+yahoofbuyv.weebly.com
+attonlineservicexx.weebly.com
+julyemailclosurer.weebly.com
```

Comment:
`pyfunceble -d julyemailclosurer.weebly.com attonlineservicexx.weebly.com yahoofbuyv.weebly.com`

You are using the Beta version of PyFunceble 4.0.0!
Please take the time to communicate with us when you notice
something unusual.




Subject                                                                                              Status      Source    
---------------------------------------------------------------------------------------------------- ----------- ----------
`attonlineservicexx.weebly.com`                                                                        INACTIVE    SPECIAL   
`julyemailclosurer.weebly.com`                                                                         INACTIVE    SPECIAL   
`yahoofbuyv.weebly.com `                                                                               INACTIVE    SPECIAL